plugins/l4d2/l4d2.cpp

```diff
@@ -1,6 +1,12 @@
-/* Copyright (C) 2005-2012, Thorvald Natvig <thorvald@natvig.com>
+/* Copyright (C) 2012-2016, Joel Kees <joelkees@gmail.com>
```

Comment:
Hi,

We're in the process of changing our license headers.
Ideally, we want all files to carry the simpler one from LICENSE.header that you have included below.
But we aren't there yet. In this case, we'd need d-rez's permission, because of the BSD license.

Here's what I suggest we do in this file, for now:

- You can add yourself to the copyright line, no problem. (But please change "2012-2016" to be just "2016" -- that's correct, right?)
- Please remove the "Mumble Developers" bit.
- Can we get a statement from you that you allow us to remove your name from ANY headers in Mumble, in order to replace the license header in the file with the simplified header? Instead of being in the header, you'll be added to the AUTHORS file, automatically generated added based on Git history. The AUTHORS file will soon be visible in both Murmur (murmur --authors for AUTHORS, murmur --license for LICNESE) and Mumble (in the about dialog).

If we have your statement on file, we'll take care of migrating the headers in the future, once people have agreed. :-)

Personal note: In practice, we can change most of our headers already. There are only a few throughout the tree where we cannot.